lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
This button doesn't include a role attribute. Should it have something for that? I defer to your accessibility expertise!